libs/driver/in-memory/src/cart/in-mem/cart.testing.service.ts

```diff
@@ -65,4 +67,8 @@ export class DaffInMemoryCartTestingService implements InMemoryDbService {
     cartItem.qty = reqBody.qty;
     this.cart.items.push(cartItem);
   }
+
+  private clearCart(): void {
+    this.cart.items = [];
```

Comment:
This should return a brand new cart instance, not one with no items.

Reply:
done